ddl/serial_test.go

```diff
@@ -53,6 +53,9 @@ import (
 // Make it serial because config is modified in test cases.
 var _ = SerialSuites(&testSerialSuite{})
 
+// TODO(tangenta): Move all the parallel tests out of this file.
+var _ = Suite(&testIntegrationSuite9{&testIntegrationSuite{}})
```

Comment:
Shall we resolve this TODO now? I think it's in the way of #26022 and we'd really need to split all these huge test files under `ddl` package.